frontend/src/components/organization/OrganizationPreviews.js

```diff
@@ -67,7 +67,7 @@ export default function OrganizationPreviews({
       {parentHandlesGridItems
         ? organizations && organizations.length > 0
           ? toOrganizationPreviews(organizations)
-          : "No Results"
+          : "No organizations found."
```

Comment:
Makes a lot of sense. Maybe it would be even better to put "No organizations found. Try changing or removing your filter or search query.". However, I'm not sure if this message makes sense for all use-cases because I don't know if a filter/search without results is the only case in which this is shown.